agent/app/agent.go

```diff
@@ -408,7 +408,7 @@ func (agent *ecsAgent) doStart(containerChangeEventStream *eventstream.EventStre
 	deregisterInstanceEventStream := eventstream.NewEventStream(
 		deregisterContainerInstanceEventStreamName, agent.ctx)
 	deregisterInstanceEventStream.StartListening()
-	taskHandler := eventhandler.NewTaskHandler(agent.ctx, agent.dataClient, state, client)
+	taskHandler := eventhandler.NewTaskHandler(agent.ctx, agent.dataClient, state, client, agent.cfg, 10*time.Minute)
```

Comment:
Took me a lot of code diving to figure out what this `10*time.Minute` actually does here. Could you make this a constant with an informative name? IIUC something along the lines of `const disconnectedModeTaskEventRetryDelay time.Duration = 10*time.Minute`